config/test.js

```diff
@@ -1,4 +0,0 @@
-module.exports = {
-  "ghUsername": "",
-  "ghPassword": ""
-};
```

Comment:
Why delete this file?

Reply:
Well, I added the test username and password to the default.json file because I didn't know the whole priority list of config files and I thought this file is useless so I deleted it. Should I add it back? Or should I just work with the local.json  and default.json file ?

Reply:
If I as a third person had to add my own credentials for these tests, how will I understand the structure of object containing all the config variables? You've added default.json to gitignore.

We need default.json, it'll contain blank strings for values but it'll be used to maintain the hierarchy of the config variables (if any hierarchy). That file will not be touched by any developer unless they want to add/rename/remove any config variable.

The developers who want to set this repo up, will have to create local.json, copy paste default.json and override the variables with their credentials. `local.json` will be in gitignore and not default.json 🙂

Reply:
Ohhh I see.. Thank you for the clarification! I will make these changes asap.